tests/plugins/elementspath/manual/elementspath.html

```diff
@@ -0,0 +1,82 @@
+<textarea id="editor1">
+	<p>Hello world. <strong>Bold text and <em>italic with additional <u>underline</u></em></strong>.</p>
+	<p>Hello world. <strong contenteditable="false">Bold text and <em contenteditable="true">italic with additional <u>underline</u></em></strong>.</p>
```

Comment:
This line looks identical as the previous line in the editor, however it has an inline widget. I would change the text so it will be more clear that there is a widget.

Reply:
> italic with additional underline inside inline widget